src/webui/master/static/js/app.js

```diff
@@ -88,6 +88,14 @@
         return state.substring(5);
       };
     })
+    .filter('taskHealth', function() {
+      return function(health) {
+        if (health != null) {
+           return health ? "healthy" : "unhealthy";
```

Comment:
2 space indent, would also be good to warn the reader here that this value is relied on for the CSS